.devcontainer/devcontainer.json

```diff
@@ -9,7 +9,7 @@
       // Append -bullseye or -focal to pin to an OS version.
       "VARIANT": "9.0",
       // Options
-      "NODE_VERSION": "16"
```

Comment:
We've tended to stick with LTS because it's a little more stable. Latest LTS is 22.

Reply:
Since I needed to update it, I just thought go all the way. Will downgrade.